packages/sitecore-jss/src/personalize/utils.ts

```diff
@@ -106,19 +106,40 @@ export class CdpHelper {
   }
 
   /**
-   * Gets the content id for CDP in the required format `embedded_[<scope>_]<id>_<lang>`
+   * Gets the friendly id for (page-level) Embedded Personalization in the required format `embedded_[<scope>_]<id>_<lang>`
    * @param {string} pageId the page id
    * @param {string} language the language
    * @param {string} [scope] the scope value
-   * @returns {string} the content id
+   * @returns {string} the friendly id
    */
-  static getContentId(pageId: string, language: string, scope?: string): string {
+  static getPageFriendlyId(pageId: string, language: string, scope?: string): string {
     const formattedPageId = pageId.replace(/[{}-]/g, '');
     const formattedLanguage = language.replace('-', '_');
     const scopeId = scope ? `${this.normalizeScope(scope)}_` : '';
     return `embedded_${scopeId}${formattedPageId}_${formattedLanguage}`.toLowerCase();
   }
 
+  /**
+   * Gets the friendly id for Component A/B Testing in the required format `component_[<scope>_]<pageId>_<componentId>_<language>*`
```

Comment:
POC had comments on using pageID here. Will this come later or is it no longer required?

Reply:
It's there already: `component_[<scope>_]**<pageId>_**<componentId>_<language>*`